transformers4rec/torch/utils/data_utils.py

```diff
@@ -367,6 +368,34 @@ def __init__(
         self.schema = schema
         self.max_sequence_length = max_sequence_length
 
+    @staticmethod
+    def _get_pad_fn(padding_lengths):
+        def pad_fn(x, y):
+            new_x = pad_batch(x, padding_lengths)
+            new_y = pad_batch(y, padding_lengths)
+            return new_x, new_y
+
+        return pad_fn
+
+    @staticmethod
+    def _augment_schema(
```

Comment:
Moved this from `schema_utiils` to here so that it's closer to the only place it's called in the codebase.